BladeSDK/src/main/java/io/bladewallet/bladesdk/Utils.kt

```diff
@@ -66,18 +68,28 @@ internal suspend fun getRemoteConfig(network: String, dAppCode: String, sdkVersi
     }
 }
 
-internal suspend fun getVisitorId(apiKey: String, context: Context): String = suspendCoroutine { continuation ->
+internal suspend fun getVisitorId(config: RemoteConfig, context: Context): String = suspendCoroutine { continuation ->
     val factory = FingerprintJSFactory(context)
-    val configuration = Configuration(
-        apiKey= apiKey,
-        region = Configuration.Region.EU,
-        endpointUrl = "https://identity.bladewallet.io"
+
+    // try once with remote config subdomain, otherwise try on default region
+    var configuration = Configuration(
+        apiKey= config.fpApiKey,
```

Comment:
Region is not needed?

Reply:
no. Getting same results as when using region US. Other regions throws "wrong region" error. So I decided not to hardcode region in case it will be changed